### PR TITLE
Initial config for next js

### PR DESCRIPTION
**Pull request by jito-jito**
## Description

Includes package **'Web-client'** in the monolith with a configuration in Next Js. 

This package replaces the previous pull request for don't have conflicts with the work between users and stalinks team.

Additionally add dependences for use Jest and React Testing Library.

# (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by gh0stl1m — DISMISSED** [review]
LGTM

File: packages/web-client/pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { createTheme } from '@platzily-ui/styling';
+import { ThemeProvider } from '@platzily-ui/styling';
+
+
+export const theme = createTheme();
+
+
+function app({ Component, pageProps }) {
```
Comment (omarefg) [inline packages/web-client/pages/_app.js]: Lets change this to App instead please. Component names should always starts with camel case.

**Review by omarefg — CHANGES_REQUESTED** [review]

File: packages/web-client/pages/_app.js (hunk 1 above)
Comment (jito-jito) [reply packages/web-client/pages/_app.js]: Yes no problem, ready 👍.

**Review by omarefg — DISMISSED** [review]

File: packages/web-client/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "react-dom": "17.0.2"
+    
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.15.0",
+    "@testing-library/react": "^12.1.2",
+    "babel-jest": "^27.3.1",
+    "eslint-config-next": "12.0.3",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^27.3.1",
```
Comment (jonattan-infante) [inline packages/web-client/package.json]: Delete jest for this packeges.json because this dependency is in the global context @jito-jito

**Review by jonattan-infante — APPROVED** [review]

**Review by omarefg — APPROVED** [review]

**Pull request merged**